client/packages/host/src/Admin/Settings.tsx

```diff
@@ -77,17 +75,7 @@ export const Settings: React.FC = () => {
         <LabelPrinterSettings />
         <ElectronSettings />
       </SettingsSection>
-      {Environment.FEATURE_PRINTER_SETTINGS && (
```

Comment:
Ability to add printers was behind a feature flag, but it was just a dummy component. Hasn't been touched for 6mo. 

I think fair to just take out at this stage? Can be added if/when we actually implement this?